pkg/server/service/proxy.go

```diff
@@ -81,6 +83,7 @@ func buildProxy(passHostHeader *bool, responseForwarding *dynamic.ResponseForwar
 		Transport:     roundTripper,
 		FlushInterval: time.Duration(flushInterval),
 		BufferPool:    bufferPool,
+		ErrorLog:      stdlog.New(log.WithoutContext().WriterLevel(logrus.ErrorLevel), "", 0),
```

Comment:
```suggestion
		ErrorLog:      stdlog.New(log.WithoutContext().WriterLevel(logrus.DebugLevel), "", 0),
```

We think that the potential `httputil.ReverseProxy` error logs are additional information to the current error handling already configured, for that reason logging as debug should be enough.